### ISSUE TITLE
G1TExtract cannot extract G1T's from the PSVita versions of the game

### ISSUE THREAD

**Issue by KuromeSan** [AdmiralCurtiss/HyoutaTools#6]
*somehow* the PC g1t files work on PSVita though which is weird since the vita certainly cannot do DX5 (see below:)
![image](https://user-images.githubusercontent.com/39113159/55204990-e88fe700-5235-11e9-9cea-8c0df222ecc3.png)

the only issue is the images are way to big for the console 

im using A20 (Atelier Nelke) 
Oh and you have to specify 0x5B as DXT5 to even get the images from A20.
``case 0x5B: Format = Textures.TextureFormat.DXT3; BitPerPixel = 8; break;``
[Textures.zip](https://github.com/AdmiralCurtiss/HyoutaTools/files/3020826/Textures.zip)

 ..-..

**Comment by AdmiralCurtiss** [comment]
I've seen your issue but I haven't gotten around to investigating this yet. Probably just something non-standard in the textures, maybe swizzling or similar?

**Comment by AdmiralCurtiss** [comment]
Yeah after taking a look these definitely look swizzled to me. Unsure what format, but I'm sure you can find information on Vita-native swizzling online somewhere...
